### ISSUE TITLE
Resizing Images by dragging corners not available

### ISSUE THREAD

**Issue by da-kicks-87** [ckeditor/strapi-plugin-ckeditor#43]
When placing an image in the editor it is not possible to resize it by dragging the corners on the image. The square indications on the corners are also not showing. 

Is there a way to turn on this functionality?

**Comment by Mgsy** [comment]
Hi! The image resize is available in the [rich preset](https://github.com/ckeditor/strapi-plugin-ckeditor/blob/master/admin/src/components/CKEditorInput/Configurator.js#L193). Could you please check if it works for your after configuring your custom field to use CKEditor 5 with the rich preset?

**Comment by da-kicks-87** [comment]
Yes it work when choosing Rich preset. I was previously using the Light version.
Is it possible to add the image resizing to all versions?
